interface-definitions/system-login.xml.in

```diff
@@ -259,6 +259,25 @@
                   </constraint>
                 </properties>
               </leafNode>
+              <leafNode name="security-mode">
+                <properties>
+                  <help>Security mode for TACACS+ authentication</help>
+                  <completionHelp>
+                    <list>mandatory optional</list>
+                  </completionHelp>
+                  <valueHelp>
+                    <format>mandatory</format>
+                    <description>Deny access immediately if TACACS+ answers with REJECT</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>optional</format>
+                    <description>Pass to the next authentication method if TACACS+ answers with REJECT</description>
+                  </valueHelp>
+                  <constraint>
+                    <regex>(mandatory|optional)</regex>
+                  </constraint>
```

Comment:
Can we make one of both a default setting to reflect the "old" behavior that was optional?

Reply:
First I wanted to say "It will require too much rewriting in `system-login.py`", but then found `from_defaults()`.

Done :)